README.rst

```diff
@@ -25,45 +25,44 @@ What is this plugin?
 
 A Perseus renderer wrapper for Kolibri that can track learning progress and save to the database.
 
-How to publish to PyPi?
+How can I install this plugin?
 ------------------------------
 
-1. Download this repo.
-2. Terminal move to the root level of repo dir and run the following commands:
-
-    ``git submodule update --init --recursive``
-
-    ``./update_perseus.sh``
-
-3. update `setup.py` to a newer version.
-4. Terminal move to `exercise_perseus_render` folder and run ``npm install``.
-5. Terminal move to the root level of repo dir and run the following commands to publish to PyPi:
+1. Inside your Kolibri virtual environment:
+    ``pip install kolibri-perseus-exercise-plugin``
 
-    ``python setup.py register -r pypi``
+2. Activate the plugin:
 
-    ``python setup.py sdist upload -r pypi``
+    ``kolibri plugin exercise_perseus_renderer enable``
 
+3. Restart Kolibri.
 
-How can I install this plugin?
+How can I install this plugin for development?
 ------------------------------
 
 1. Download this repo.
-2. Terminal move to the repo dir and run the following commands:
+2. Open terminal in your Kolibri repo.
+4. run the following commands:
 
-    ``git submodule update --init --recursive``
+    ``pip install -e <KOLIBRI-PERSEUS-PLUGIN-LOCAL-PATH>``
 
-    ``./update_perseus.sh``
+    ``kolibri plugin exercise_perseus_renderer enable``
 
-3. Terminal move to your Kolibri dir
-4. run the following commands:
+5. Then run the commands to install frontend packages in Kolibri, this plugin will have its recursively installed.
```

Comment:
I'm also unable to complete this command. It hangs indefinitely with no CPU or network activity at `[2/4] Fetching packages...`:

```
(le)~/P/l/r/kolibri (brand) $ yarn install
yarn install v0.19.1
$ node ./frontend_build/src/npm_deprecation_warning.js
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ node ./frontend_build/src/install_dependencies.js
INFO     Writing webpack_json output to /var/folders/n9/k997qvqs3f7_m_310crwsxyc0000gp/T/117112-42322-biev48.my56sqncdi.json
INFO:kolibri.core.webpack.management.commands.webpack_json:Writing webpack_json output to /var/folders/n9/k997qvqs3f7_m_310crwsxyc0000gp/T/117112-42322-biev48.my56sqncdi.json
Installing kolibri/plugins/coach_tools/package.json...
yarn install v0.19.1
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.11s.

Installing kolibri/plugins/audio_mp3_render/package.json...
yarn install v0.19.1
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.31s.

Installing kolibri/plugins/document_pdf_render/package.json...
yarn install v0.19.1
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.31s.

Installing /Users/d/Projects/learning_equality/repos/kolibri-exercise-perseus-plugin/exercise_perseus_renderer/package.json...
yarn install v0.19.1
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
$ git submodule update --init --recursive && cd node_modules/perseus && yarn install
Submodule path 'node_modules/perseus': checked out '55e99dd019440ef72418fc74321111ddbadedf74'
Submodule path 'node_modules/perseus/hubble': checked out '2f376e8f9fd0a67d03c047c5057c402fba6447e5'
Submodule path 'node_modules/perseus/kmath': checked out '92454b19c2ca7f8c54f61a6ff5c2d7789cef9631'
Submodule path 'node_modules/perseus/lib/mathjax': checked out 'ae516f597892a400abff79b73cecb460ac18501b'
Submodule path 'node_modules/perseus/math-input': checked out '2f4a70ffb1acd41845d5f0f39fae33da2bab33e9'
Submodule path 'node_modules/perseus/react-components': checked out '777fd2ff24fa0af33237d5b28039729534155c23'
Submodule path 'node_modules/perseus/react-components/react-live-editor': checked out 'a050794677ea55f904ff4b6ed9d646223975750d'
Submodule path 'node_modules/perseus/simple-markdown': checked out '27d1bc0a3efecd34a35f0da0659db50dcc4f49fc'
yarn install v0.19.1
info No lockfile found.
warning perseus@0.0.1: "dependencies" has dependency "aphrodite" with range "^0.3.0" that collides with a dependency in "devDependencies" of the same name with version "^0.3.1"
[1/4] Resolving packages...
warning jsx-loader > react-tools@0.11.2: react-tools is deprecated. For more information, visit https://fb.me/react-tools-deprecated
warning jsx-loader > react-tools > commoner > graceful-fs@2.0.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
warning mocha > glob > minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning mocha > glob > graceful-fs@2.0.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
warning jsx-loader > react-tools > commoner > glob > minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning mocha > jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
warning jest-cli > istanbul > fileset > minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning less > request > node-uuid@1.4.7: use uuid module instead
warning webpack > watchpack > chokidar > readdirp > minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[2/4] Fetching packages...
```